src/validators/matchesValue.js

```diff
@@ -0,0 +1,13 @@
+import createValidator from '../createValidator';
+
+export default function matchesValue(matchKey, matchLabel) {
```

Comment:
I think I like the name `matchesField` better and maybe renaming the parameters to `otherFieldName` and `otherFieldLabel`. `matchesValue` felt a little ambiguous to me at first.